index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@Sir_Malcom</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section class="parent1">
+      <nav class="dcp1 navbar">
+          <img class="logo" src="./elements/lg.png" alt="Logo">
+          <div></div>
+          <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
+      </nav>
+      <div class="space"></div>
+      <article class="dcp1 art1">
+          <h4>
+        I’m Sir_Malcom
+        Glad to see you!
+          </h4>
```

Comment:
- Please replace the `<h4>` tag with an `<h1>` tag, so as to have a well structured semantically correct HTML code
- Please the sentence ` I’m Sir_Malcom Glad to see you!`, make use of the correct font family which is `Poppins` and you should employ a `<br>` tag in the appropriate place, so as to make your project design more similar to the Figma design.
- Please use the appropriate spacing (padding and margins) as described in the figma design. 
##### Figma design
![Screenshot from 2022-05-27 02-58-34](https://user-images.githubusercontent.com/58771507/170613475-a627f990-f0c9-4c59-8d6c-6f9f6d6c8828.png)

##### Your design
![Screenshot from 2022-05-27 02-59-34](https://user-images.githubusercontent.com/58771507/170613583-a550ee4c-39b4-40a0-9544-10d7ddf2b364.png)